01_fibonacci/main_test.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestPrintNumber(t *testing.T) {
+	r := require.New(t)
+
+	r.Equal(1, printNumber(2))
+	r.Equal(2, printNumber(3))
+}
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n21\n34\n")
```

Comment:
https://golang.org/pkg/strconv/#example_Quote

`strconv.Quote` wraps the given string in double quotes. Since you are wrapping both `expected` and `actual` values in quotes you can completely get rid of it and directly compare the two string.

Reply:
Thank you !
removed "strconv.Quote"